_pages/en/strategy-build-vs-buy.md

```diff
@@ -280,7 +290,8 @@ Based on the risk level, ESDC will adjust its decisions around acquiring softwar
 Risk levels are heavily influenced by the capability being enabled by software.
 The following diagram helps visualize how this risk level varies according with the type of capabilities:
 
-![A 3-layered circle showing that risk increases from exterior to interior circles, the most exterior one being Productivity, the middle one being Corporate or Shared, and the innermost being Core to Mandate](assets/images/strategy-build-buy-risk-circle.png){:height="50%" width="50%" style="display:block;margin-left:auto;margin-right:auto"}
+![A 3-layered circle with an arrow pointing from the external circle crossing all the way towards the internal circle.
+The external circle is called "Productivity", the middle circle is called "Corporate or Shared" and the internal circle is called "Core to Mandate"](assets/images/strategy-build-buy-risk-circle.png){:height="50%" width="50%" style="display:block;margin-left:auto;margin-right:auto"}
```

Comment:
maybe change to "A 3-layered circle. The external circle is called "Productivity", the middle circle is called "Corporate or Shared" and the internal circle is called "Core to Mandate". There is an arrow that starts at the external circle which points towards the inner circle and has the text "Risk increases" on it.

Reply:
Thanks! I’ll make the change!